test/3nodetest/Makefile

```diff
@@ -20,7 +19,8 @@ data/filerand: bin/random
 	./bin/random 50000000 > ./data/filerand
 
 bin/random: $(RANDOMSRC)/**/*.go
-	go build -o ./bin/random $(RANDOMSRC)/random
+	go get github.com/jbenet/go-random
+	go build -o ./bin/random github.com/jbenet/go-random/random
```

Comment:
This is the only real change. It uses go-get, not gx, but, as far as I can tell, we don't use these tests at the moment anyways.

Reply:
Isn't this also https://github.com/ipfs/go-ipfs/pull/5440/files#diff-c8660427d0699f8fe97cb9943c5bd91eR40? I'd just depend on that